### PR TITLE
Capitalize roll numbers of students on the leaderboard

### LINKED ISSUE #16: Capitalize roll numbers of students on the leaderboard
Currently, the roll numbers of users on the leaderboard are displayed with different capitalizations. For example, one of the students has a roll number "2020bcs-079" and the others have "2020BCS-036" in uppercase. There may also be roll numbers in mixed cases. This can be easily fixed by changing the roll number to uppercase.

### PR DESCRIPTION
**Pull request by AmanPachori**
@hs2361 I have Capitalize roll numbers of students on the leaderboard. Fixes #16

### CODE REVIEW

File: src/components/LeaderboardTile.svelte [hunk 1]
```diff
@@ -8,6 +8,7 @@
     let components = l.split('/')
     return components[components.length - 1]
   }
+  r.rollNumber=r.rollNumber.toUpperCase();
```
Comment (hs2361) [inline src/components/LeaderboardTile.svelte:11]: Could you please add in spaces before and after the `=` operator?

**Review by hs2361 — CHANGES_REQUESTED** [review]

**Comment by hs2361** [comment]
@AmanPachori Please add a more descriptive PR description. Please also reference the issue you're solving with this PR (such as "closes" or "fixes") so that the issue gets closed automatically.

**Comment by AmanPachori** [comment]
@hs2361 I have done the changes that were requested

**Review by hs2361 — APPROVED** [review]

**Comment by hs2361** [comment]
@AmanPachori Please update the PR description and write the issue number this PR will close. Your changes LGTM. Please squash your commits into one.

**Pull request merged**